react/react1/week2/toDoApp/src/components/Todo/Todo.styled.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export const Label = styled.label`
```

Comment:
There are other ways to make our style adaptive that might be a bit easier. I'd have used inline style in this instance

Reply:
I thought about inline styles already after the pull request.🙈 Should I change it to inline? Or can I merge the branch?

Reply:
This also works, so it's up to you :)

Reply:
Ok. Thank you!